backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -217,7 +225,8 @@ def sanitize_records(self, records, selected_trees, filters):
                     tests_or_boots["archSummary"][archKey] = archSummary
                 archSummary["status"][status] += 1
 
-            if build_id not in processed_builds:
+            build_is_not_processed = build_id not in processed_builds
```

Comment:
nit: It is good practice to keep flag names in positive format, such as `is_build_precesses`

Reply:
Done